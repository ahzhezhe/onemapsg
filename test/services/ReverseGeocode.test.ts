import { oneMap } from '..';

test('revgeocodexy', async () => {
  const result = await oneMap.reverseGeocode.revgeocodexy({
    location: [28833.0605447, 31810.3242714]
  });
  expect(result).toMatchSnapshot();
});

test('revgeocode', async () => {
  const result = await oneMap.reverseGeocode.revgeocode({
    location: [1.3039454, 103.8408471]
  });
  expect(result).toMatchSnapshot();
});
