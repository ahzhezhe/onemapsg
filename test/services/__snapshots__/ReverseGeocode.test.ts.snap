// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`revgeocode 1`] = `
{
  "GeocodeInfo": [
    {
      "BLOCK": "65",
      "BUILDINGNAME": "WATERSCAPE AT CAVENAGH",
      "LATITUDE": "1.303955777623504",
      "LONGITUDE": "103.84080401483146",
      "POSTALCODE": "229619",
      "ROAD": "CAVENAGH ROAD",
      "XCOORD": "28833.0605447",
      "YCOORD": "31810.3242714",
    },
  ],
}
`;

exports[`revgeocodexy 1`] = `
{
  "GeocodeInfo": [
    {
      "BLOCK": "65",
      "BUILDINGNAME": "WATERSCAPE AT CAVENAGH",
      "LATITUDE": "1.303955777623504",
      "LONGITUDE": "103.84080401483146",
      "POSTALCODE": "229619",
      "ROAD": "CAVENAGH ROAD",
      "XCOORD": "28833.0605447",
      "YCOORD": "31810.3242714",
    },
  ],
}
`;
